### PR TITLE
Pin browser tests to Chrome 127

### PR DESCRIPTION
**Pull request by georgegevoian**
## Context

A handful of browser tests began failing in Chrome 128+ (#1185), which the CI workflow began using recently.

The root cause appears to be a change in alert behavior when running automated tests: alerts are no longer shown, even when the BiDi Selenium driver is used.

## Proposed solution

Pin Chrome to version 127 when running any browser tests. Follow up later with a fix or workaround.

## Has this been tested?

- [ ] 👍 yes, I added tests to the test suite
- [ ] 💭 no, because this PR is a draft and still needs work
- [x] 🙅 no, because this is not relevant here
- [ ] 🙋 no, because I need help

### CODE REVIEW

File: app/client/ui/PageWidgetPicker.ts [hunk 1]
```diff
@@ -260,8 +261,7 @@ export function buildPageWidgetPicker(
     dom.create(PageWidgetSelect,
       value, tables, columns, onSaveCB, behavioralPromptsManager, options),
 
-    // gives focus and binds keydown events
-    (elem: any) => { setTimeout(() => elem.focus(), 0); },
+    elem => { FocusLayer.create(ctl, {defaultFocusElem: elem, pauseMousetrap: true}); },
```
Comment (berhalak) [inline app/client/ui/PageWidgetPicker.ts:264]: Is this needed here?

Comment (georgegevoian) [reply app/client/ui/PageWidgetPicker.ts:264]: > Is this needed here?

[This test](https://github.com/gristlabs/grist-core/actions/runs/10641941256/job/29503756696) was failing regularly without it.

**Review by berhalak — APPROVED** [review]
Thanks.

File: test/test_env.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+export GRIST_SESSION_COOKIE="grist_test_cookie"
+export LANGUAGE="en_US"
+export SE_BROWSER="chrome"
+export SE_BROWSER_VERSION="127"
+export SE_DRIVER="chrome-driver"
+export SE_DRIVER_VERSION="127.0.6533.119"
+export TEST_CLEAN_DATABASE="true"
+export TEST_SUPPORT_API_KEY="api_key_for_support"
+
+exec "$@"
```
Comment (jordigh) [inline test/test_env.sh:12]: I like the cleanup to have this all in one place, and I'm happy too about pinning the Chrome version.

As a follow-up, we should have some automation to increase the version number with new releases and automatically merge in a new version number if the CI passes.

**Review by jordigh — APPROVED** [review]

**Pull request merged**